src/dolphin/unwrap/_unwrap_3d.py

```diff
@@ -108,3 +118,29 @@ def _map_date_str_to_file(
     # None is special case for reference epoch
     date_str_to_file[first_date] = None
     return date_str_to_file
+
+
+def _create_conncomps_from_mask(
+    temporal_coherence_file: PathOrStr,
+    temporal_coherence_threshold: float,
+    unw_filenames: Sequence[PathOrStr],
+) -> list[Path]:
+    arr = io.load_gdal(temporal_coherence_file, masked=True)
+    # XX: for now, "connected component" files are
```

Comment:
"XXX" is the universally agreed-upon way to spell this 😝

https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments

```suggestion
    # XXX: for now, "connected component" files are
```